coalib/bearlib/__init__.py

```diff
@@ -7,8 +7,12 @@
 import logging
 from functools import wraps
 
+import pdb
+import colorit
 from coalib.settings.FunctionMetadata import FunctionMetadata
 
+db = pdb.Pdb()
+db.prompt = colorit.color_front('(pdb)',50,200,50)
```

Comment:
Coloring shouldn't be done by default. I mean I would even prefer to have a powerline prompt here :P
Such design considerations should be shifted to the end of implementation, as this depends on your terminal used (This could really make problems with Windows)